packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/interactive-dark-theme" as *;
+
+// ----------------
+
+// TODO: Create app-header tokens:
+:root {
+  --token-app-header-height: 60px;
+  --token-app-header-home-link-padding: 4px;
```

Comment:
I will add these tokens to JSON but wanted to confirm if naming pattern I used is correct?